web/src/filters.ts

```diff
@@ -0,0 +1,63 @@
+// Types and parsing logic for log filters.
```

Comment:
Can this file be logfilters.ts? I'm working on sidebar resource type filters, so the disambiguation helps.

Reply:
changed to logfilters.ts!